### PR TITLE
edit wrong named enves

### LINKED ISSUE #7292: Wrong named envvars
Just identified when looking into [global envvars](https://doc.owncloud.com/ocis/next/deployment/services/env-vars-special-scope.html) that should start with `OCIS_`

`STORAGE_USERS_PERMISSION_ENDPOINT` --> `OCIS_STORAGE_USERS_PERMISSION_ENDPOINT`

`STORAGE_USERS_ASYNC_PROPAGATOR_PROPAGATION_DELAY` --> `OCIS_STORAGE_USERS_ASYNC_PROPAGATOR_PROPAGATION_DELAY`

In the **graph** service, same as in userlog, there is the key-pair:
`OCIS_MACHINE_AUTH_API_KEY`
`USERLOG_MACHINE_AUTH_API_KEY` --> `GRAPH_MACHINE_AUTH_API_KEY`

`WEB_UI_CONFIG_FILE` is double present, can be seen in the web service too, we need to check why.

Following envvars are used in both the `userlog` and `clientlog` service. Those should be then either a global one starting with `OCIS_` or the clientlog version adapted to `CLIENTLOG_`:

`USERLOG_DEBUG_ADDR`
`USERLOG_DEBUG_PPROF`
`USERLOG_DEBUG_TOKEN`
`USERLOG_DEBUG_ZPAGES`
`USERLOG_LOG_COLOR`
`USERLOG_LOG_FILE`
`USERLOG_LOG_LEVEL`
`USERLOG_LOG_PRETTY`

@kobergj @dragonchaser

### PR DESCRIPTION
**Pull request by case0sh**
## Description
Checked and changed the envvars specified in the task and also removed those that are no longer used.

## Related Issue
- Fixes #7292

## How Has This Been Tested?
- test environment:
- test case 1:

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:
- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: services/auth-machine/pkg/config/config.go [hunk 1]
```diff
@@ -20,7 +20,6 @@ type Config struct {
 
 	SkipUserGroupsInToken bool `yaml:"skip_user_groups_in_token" env:"AUTH_MACHINE_SKIP_USER_GROUPS_IN_TOKEN" desc:"Disables the encoding of the user's group memberships in the reva access token. This reduces the token size, especially when users are members of a large number of groups."`
 
-	MachineAuthAPIKey string `yaml:"machine_auth_api_key" env:"OCIS_MACHINE_AUTH_API_KEY;AUTH_MACHINE_API_KEY" desc:"Machine auth API key used to validate internal requests necessary for the access to resources from other services."`
```
Comment (kobergj) [inline services/auth-machine/pkg/config/config.go:23]: Sorry. Seems that was overseen in the ticket. `auth-machine` still needs the `MachineAuthAPIKey` please readd it

File: services/clientlog/pkg/config/debug.go [hunk 2]
```diff
@@ -2,8 +2,8 @@ package config
 
 // Debug defines the available debug configuration.
 type Debug struct {
-	Addr   string `yaml:"addr" env:"USERLOG_DEBUG_ADDR" desc:"Bind address of the debug server, where metrics, health, config and debug endpoints will be exposed."`
-	Token  string `yaml:"token" env:"USERLOG_DEBUG_TOKEN" desc:"Token to secure the metrics endpoint."`
-	Pprof  bool   `yaml:"pprof" env:"USERLOG_DEBUG_PPROF" desc:"Enables pprof, which can be used for profiling."`
-	Zpages bool   `yaml:"zpages" env:"USERLOG_DEBUG_ZPAGES" desc:"Enables zpages, which can be used for collecting and viewing in-memory traces."`
+	Addr   string `yaml:"addr" env:"CLIENTLOG_USERLOG_DEBUG_ADDR" desc:"Bind address of the debug server, where metrics, health, config and debug endpoints will be exposed."`
```
Comment (kobergj) [inline services/clientlog/pkg/config/debug.go:5]: Should be `CLIENTLOG_DEBUG_ADDR`. Remove the `USERLOG` (for all envvars in clientlog/ folder)

File: services/storage-users/pkg/config/config.go [hunk 3]
```diff
@@ -94,7 +94,7 @@ type Drivers struct {
 
 // AsyncPropagatorOptions configures the async propagator
 type AsyncPropagatorOptions struct {
-	PropagationDelay time.Duration `yaml:"propagation_delay" env:"STORAGE_USERS_ASYNC_PROPAGATOR_PROPAGATION_DELAY" desc:"The delay in seconds between a change made to a tree and the propagation start on treesize and treetime. Multiple propagations are computed to a single one."`
+	PropagationDelay time.Duration `yaml:"propagation_delay" env:"OCIS_STORAGE_USERS_ASYNC_PROPAGATOR_PROPAGATION_DELAY" desc:"The delay in seconds between a change made to a tree and the propagation start on treesize and treetime. Multiple propagations are computed to a single one."`
```
Comment (kobergj) [inline services/storage-users/pkg/config/config.go:97]: Not needed. See https://github.com/owncloud/ocis/issues/7292#issuecomment-1722875657 please revert

File: services/storage-users/pkg/config/config.go [hunk 4]
```diff
@@ -105,7 +105,7 @@ type OCISDriver struct {
 	// Root is the absolute path to the location of the data
 	Root                string `yaml:"root" env:"STORAGE_USERS_OCIS_ROOT" desc:"The directory where the filesystem storage will store blobs and metadata. If not defined, the root directory derives from $OCIS_BASE_DATA_PATH:/storage/users."`
 	UserLayout          string `yaml:"user_layout" env:"STORAGE_USERS_OCIS_USER_LAYOUT" desc:"Template string for the user storage layout in the user directory."`
-	PermissionsEndpoint string `yaml:"permissions_endpoint" env:"STORAGE_USERS_PERMISSION_ENDPOINT;STORAGE_USERS_OCIS_PERMISSIONS_ENDPOINT" desc:"Endpoint of the permissions service. The endpoints can differ for 'ocis' and 's3ng'."`
+	PermissionsEndpoint string `yaml:"permissions_endpoint" env:"OCIS_STORAGE_USERS_PERMISSION_ENDPOINT;STORAGE_USERS_OCIS_PERMISSIONS_ENDPOINT" desc:"Endpoint of the permissions service. The endpoints can differ for 'ocis' and 's3ng'."`
```
Comment (kobergj) [inline services/storage-users/pkg/config/config.go:108]: Not needed. See https://github.com/owncloud/ocis/issues/7292#issuecomment-1722875657 please revert

File: services/webfinger/pkg/config/config.go [hunk 5]
```diff
@@ -22,7 +22,7 @@ type Config struct {
 	Relations []string   `yaml:"relations" env:"WEBFINGER_RELATIONS" desc:"A comma-separated list of relation URIs or registered relation types to add to webfinger responses."`
 	IDP       string     `yaml:"idp" env:"OCIS_URL;OCIS_OIDC_ISSUER;WEBFINGER_OIDC_ISSUER" desc:"The identity provider href for the openid-discovery relation."`
 	OcisURL   string     `yaml:"ocis_url" env:"OCIS_URL;WEBFINGER_OWNCLOUD_SERVER_INSTANCE_URL" desc:"The URL for the legacy ownCloud server instance relation (not to be confused with the product ownCloud Server). It defaults to the OCIS_URL but can be overridden to support some reverse proxy corner cases. To shard the deployment, multiple instances can be configured in the configuration file."`
-	Insecure  bool       `yaml:"insecure" env:"OCIS_INSECURE;WEBFINGER_INSECURE" desc:"Allow insecure connections to the WEBFINGER service."`
+	Insecure  bool       `yaml:"insecure" env:"OCIS_INSECURE;WEBFINGER_INSECURE" desc:"Allow insecure connections to the OIDC provider."`
```
Comment (kobergj) [inline services/webfinger/pkg/config/config.go:25]: Is this change intended?

**Review by kobergj — CHANGES_REQUESTED** [review]

**Comment by kobergj** [comment]
There are still occurrences of `MachineAuthAPIKey` in the `idp` service. That's why it doesn't build.

File: services/clientlog/pkg/config/debug.go (hunk 2 above)
Comment (case0sh) [reply services/clientlog/pkg/config/debug.go]: ☑️

File: services/storage-users/pkg/config/config.go (hunk 3 above)
Comment (case0sh) [reply services/storage-users/pkg/config/config.go]: ☑️

File: services/storage-users/pkg/config/config.go (hunk 4 above)
Comment (case0sh) [reply services/storage-users/pkg/config/config.go]: ☑️

File: services/webfinger/pkg/config/config.go (hunk 5 above)
Comment (case0sh) [reply services/webfinger/pkg/config/config.go]: ☑️

File: services/auth-machine/pkg/config/config.go (hunk 1 above)
Comment (case0sh) [reply services/auth-machine/pkg/config/config.go:23]: ☑️

**Review by kobergj — APPROVED** [review]

**Pull request merged**